### PR TITLE
Move labels discussion back to its concept page

### PR DESCRIPTION
**Pull request by tengqm**
The discussion about "managing labels effectively" and "updating labels" are currently difficult to find. They are supposed to be part of the labels concept page. This PR moves this two sections there and adjusts the links where they were pointed at.

### CODE REVIEW

File: content/en/docs/concepts/overview/working-with-objects/labels.md [hunk 1]
```diff
@@ -297,11 +297,109 @@ One use case for selecting over labels is to constrain the set of nodes onto whi
...
+multi-tier application, such as the [guestbook example](https://github.com/kubernetes/examples/tree/master/guestbook/),
+would additionally need to distinguish each tier. The frontend could carry the following labels:
+
+```yaml
+labels:
+  app: guestbook
+  tier: frontend
+```
+
+while the Redis master and slave would have different `tier` labels, and perhaps even an
```
Comment (sftim) [inline content/en/docs/concepts/overview/working-with-objects/labels.md:316]: Eeesh. Some **other** PR should avoid the terms “master“ and “slave“; some countries still permit slavery or even officially endorse it; other places merely regret their history of involvement.

There are actual victims of slavery who find this usage unsettling.

**Review by sftim — COMMENTED** [review]
/lgtm

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

**Comment by reylejano** [comment]
Thank you, the "Using labels effectively" section is logically better in the concepts/overview/working-with-objects/labels page
/approve

**Pull request merged**